profiles/permissions.py

```diff
@@ -29,7 +29,7 @@ def has_permission(self, request, view):
 class IsOwnCompany(BasePermission):
     def has_permission(self, request, view):
         user = request.user
-        pk = request.data.get("company_pk")
+        pk = request.data.get("company")
```

Comment:
Please create a task to remove all manipulations with request payload and/or query parameters from permission classes.

Permissions must be tied ONLY to the 'core' part of URL and authenticated user.